_ml-commons-plugin/cluster-settings.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: ML Commons cluster settings
+has_children: false
+nav_order: 10
+---
+
+# ML Commons cluster settings
+
+This page provides an overview of `opensearch.yml` settings that can be configured for the ML commons plugin.
+
+| **Setting** | **Default value** | **Value range** | **Description** 
+| :--- | :--- |:--- | :--- 
+| `plugins.ml_commons.only_run_on_ml_node` | false | true or false | If `true`, ML Commons tasks and models run machine learning (ML) tasks on ML nodes only. If `false`, tasks and models run on ML nodes first. If no ML nodes exist, tasks and models run on data nodes. Don't set as "false" on production cluster. 
+| `plugins.ml_commons.task_dispatch_policy` | round_robin |  round_robin or least_load | Default is `round_robin`. `round_  robin` dispatches ML tasks to ML nodes using round robin routing. `least_load` gathers all an ML nodes' runtime information, like JVM heap memory usage and running tasks, then dispatches tasks to the ML node with the least load. 
+| `plugins.ml_commons.sync_up_job_interval_in_seconds` | 10 | [0, 86_400] | If set as 0, ML Commons immediately stops any syncup jobs. 
```

Comment:
Add more details ?
"Sync up job will regularly check running models on each node, will sync up for newly loaded or removed models."

Reply:
You can also explain some details about sync up job under profile API. Draft words:

Profile API returns runtime information. ml-commons will run regular sync up job to sync up newly added/removed models on each node. For example, if node restarted, will sync up the model removed information to all nodes.